### DISCUSSION TITLE
How better can I use the library?

### DISCUSSION THREAD

**Discussion by knightswatch3** [airbnb/visx#1152] [Q&A]
Hello I am new to this library and I am exploring a better option replacing my current (custom built graphing library on the UI).

I am choosing between visx and cystoscape.

I am curious to know if visx let me add nodes on the fly with click events like how cytoscape does?

I'd appreciate any additional information besides this.  TIA

**Comment by williaster** [comment]
This was answered in [my response to your other discussion question](https://github.com/airbnb/visx/discussions/1130#discussioncomment-591835).
